static/game/bullet.js

```diff
@@ -35,7 +35,12 @@ export class Bullet extends Entity {
     this.bounciness = 0;
     this.color = color;
     this.damage = damage;
-    /** @type {(function(Bullet): void)[]} */
+    /**
+     * @type {{ name: string
+     *        , data: number
+     *        , func: (function(Bullet, number): void)
+     *        }[]}
```

Comment:
I guess we can do the weird Haskell commas in doc comments long as Intellisense still gets what's going on and if it doesn't fit on one line. (Does this one not?)

Reply:
It doesn't fit in one line and I don't know what the JavaScript convention on these is